tests/test_reduction_ops.py

```diff
@@ -670,3 +670,19 @@ def test_accuracy_vectornorm(shape, ord, dim, keepdim, dtype):
         res_out = torch.linalg.vector_norm(inp, ord, dim, keepdim)
 
     gems_assert_close(res_out, ref_out, dtype)
+
+
+@pytest.mark.parametrize("shape", REDUCTION_SHAPES)
```

Comment:
It seem that test for masked_fill should go into pointwise operation.

Reply:
Already moved the relevant tests into the pointwise category.